src/wasm-lib/kcl/src/std/sketch.rs

```diff
@@ -1043,10 +1044,11 @@ async fn inner_start_sketch_on(
     args: &Args,
 ) -> Result<SketchSurface, KclError> {
     match data {
-        SketchData::Plane(plane_data) => {
+        SketchData::PlaneData(plane_data) => {
```

Comment:
What is the difference between `Plane` and `PlaneData`?

Reply:
A [`Plane`](https://github.com/KittyCAD/modeling-app/blob/franknoirot/4468/offset-planes-visible/src/wasm-lib/kcl/src/executor.rs#L671-L686) is a full-fledged 3D object, which [`PlaneData`](https://github.com/KittyCAD/modeling-app/blob/franknoirot/4468/offset-planes-visible/src/wasm-lib/kcl/src/std/sketch.rs#L891-L924) is more of a reference either to a named default plane or the origin and axes of a custom one.

I find it a bit odd, but up until now the only custom plane definition has been as an object of this `PlaneData` in the `startSketchOn` function, not as a whole 3D entity in its own right. There has been no `plane` standard library function, for example.

Reply:
Alright I've made a few changes to make the distinction between a `Plane` and what I'm now calling `PlaneOrientationData`, which can be either a reference to default plane or the orientation info needed to construct a custom plane